src/main/java/bob/command/FindCommand.java

```diff
@@ -0,0 +1,26 @@
+package bob.command;
+
+import bob.storage.Storage;
+import bob.task.TaskList;
+import bob.ui.Ui;
+
+/**
+ * Command to find a task by description
+ */
+public class FindCommand extends Command {
+    private final String input;
```

Comment:
```suggestion
    /** User input string to search for. */
    private final String input;
```
Javadoc of class members could be added as well